### PR TITLE
Rename OpenTelemetry service to collector

### PR DESCRIPTION
**Pull request by huikang**
Signed-off-by: Hui Kang <kangh@us.ibm.com>

### CODE REVIEW

File: specification/overview.md [hunk 1]
```diff
@@ -249,19 +249,19 @@ into a binary or text format. Currently there are two types of propagators:
 
 ## Agent/Collector
```
Comment (SergeyKanzhelev) [inline specification/overview.md]: Can you please also rename this to something like: 

```suggestion
## Collector (a.k.a. Agent)
```

Comment (SergeyKanzhelev) [reply specification/overview.md]: Or simply "Collector" and mention other names in the text below

Comment (huikang) [reply specification/overview.md]: Thanks. PR is updated.

**Comment by SergeyKanzhelev** [comment]
this change is small and non controversial. Decision of a name change was announced more than a week ago. Merging without 2 business days wait.

Thank you @huikang !

**Comment by tigrannajaryan** [comment]
@huikang thank you, I missed this part when doing the renaming.

**Pull request merged**